Makefile

```diff
@@ -78,9 +78,11 @@ lint:
 
 # Generate code
 generate: controller-gen
+	./hack/kogito-module-api.sh --disable
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
-	./hack/openapi.sh
-	./hack/client-gen.sh
+	#./hack/openapi.sh
```

Comment:
Why is this line commented out?

Reply:
done by mistake while testing my changes. Reverted them